### ISSUE TITLE
FeatureSetSelector module needs to implement a mask to be consistent with sklearn

### ISSUE THREAD

**Issue by perib** [EpistasisLab/tpot2#71]
see https://scikit-learn.org/stable/modules/generated/sklearn.feature_selection.SelectorMixin.html
